server/api/reports.py

```diff
@@ -28,6 +28,16 @@ def pretty_targeted(is_targeted: bool) -> str:
     return "Targeted" if is_targeted else "Opportunistic"
 
 
+def pretty_pvalue(value: float) -> str:
+
+    if value is None:
+        return ""
+    elif value < 10 ** -10:
+        return "<0.0000000001"
```

Comment:
i know this is the most accurate, but wondering if it will make it more difficult for consumers of the audit report to use this value, since now it's not a number. thoughts on just removing the `<`?

Reply:
@mcchilders this seems like a Q for you

Reply:
In this one instance, where the number we’re trying to represent is smaller than what we can show with our 10 significant digits, I think the “less than” needs to be there. For the numbers that we can represent accurately in 10 decimals, we won’t use it. Does that make sense? This just covers the “we limited our digits and this is smaller than a number we can show.